workflows/dags/literature/hep_create_dag.py

```diff
@@ -1280,21 +1280,32 @@ def check_is_auto_approved(**context):
                 "halt_for_approval_if_new_or_reject_if_not_relevant.is_record_relevant"
             )
 
-        @task
+        @task.branch
         def is_record_relevant(**context):
             workflow_data = read_object(
                 s3_hook, bucket_name, context["params"]["workflow_id"]
             )
-            if is_submission(workflow_data):
-                return True
 
-            if is_journal_coverage_full(workflow_data):
-                return True
+            if (
+                is_submission(workflow_data)
+                or is_journal_coverage_full(workflow_data)
+                or is_auto_approved(workflow_data)
```

Comment:
for this one, lets also impelment that part https://github.com/inspirehep/inspirehep/blob/6526a989548118fdac5c9a2b01bf8e9bc01d9bf4/workflows/plugins/include/inspire/is_record_relevant.py#L14-L16